openstates/tx/bills.py

```diff
@@ -259,30 +268,34 @@ def scrape_bill(self, session, history_url):
                     ctty = ctty.replace(r, "").strip()
                 extra['committees'] = ctty
 
-            bill.add_action(actor, action.findtext('description'),
-                            act_date, type=atype, **extra)
+            bill.add_action(
+                action.findtext('description'),
+                act_date,
+                chamber=actor,
+                classification=atype
+            )
 
         for author in root.findtext('authors').split(' | '):
             if author != "":
-                bill.add_sponsor('primary', author, official_type='author')
+                bill.add_sponsorship(author, classification='primary',
+                                     entity_type='person', primary=True)
```

Comment:
have you seen any committees in the data?  do they identify themselves in any way (i.e. do they all say committee?) 

if not it might be OK to assume they're all person